### PR TITLE
Add Institute and Area relation to User

### PR DESCRIPTION
**Pull request by LDVera**
the migration to add Institution and Area references to user was created
in the admin_user views some files was modificated to implement the funtionallity
into the admin_user_controller.rb file was permitted both parameters to storage in the db

### CODE REVIEW

File: app/views/admin_users/_search.html.erb [hunk 1]
```diff
@@ -4,7 +4,16 @@
     <td><%= user.lasname %></td>
     <td><%= user.email %></td>
     <td><%= user.control_number %></td>
+    <td><%= user.institution.description%></td>
+    <td><%= user.area.description%></td>
+    <% # @institutions.each do |inst|%>
+    
+    <% # end%>
+    <% # datos = Institution.joins(:description).where("institution_id" => user.institution_id)  %>
```
Comment (LDVera) [inline app/views/admin_users/_search.html.erb:12]: 😢

**Review by LPMendoza — APPROVED** [review]
It is a good work, but you could include more features in this pull request, I think this pr is a little bit small

**Review by ArielMD — APPROVED** [review]
nice job, you already have relationships between users and institutions

**Review by mariormt17 — APPROVED** [review]
Good job with these changes!

**Pull request merged**